Mention future minset additions (#1105)
ietf-tapswg/api-drafts: draft-ietf-taps-arch.md

## Provide Common APIs for Common Features

> Any functionality that is common across multiple transport protocols SHOULD be made accessible through a unified set of calls using the Transport Services API. As a baseline, any Transport Services API SHOULD allow access to the minimal set of features offered by transport protocols {{?RFC8923}}. If that minimal set is updated and expanded in the future, the Transport Services API ought to be extended to match, if necessary.

Reviewer: ```suggestion
Any functionality that is common across multiple transport protocols SHOULD be made accessible through a unified set of calls using the Transport Services API. As a baseline, any Transport Services API SHOULD allow access to the minimal set of features offered by transport protocols {{?RFC8923}}. If that minimal set is updated and expanded in the future, the Transport Services API ought to be extended to match.
```

I just find the "if necessary"  a bit too much ... "if it happens, it ought to, if necessary". I'm not even sure what "if necessary" means here... when would doing that become necessary, and when not? What would make it necessary - compatibility, business reasons?

Reviewer 2: also: I think it's /updated **or** extended/ not both.

Author: ```suggestion
Any functionality that is common across multiple transport protocols SHOULD be made accessible through a unified set of calls using the Transport Services API. As a baseline, any Transport Services API SHOULD allow access to the minimal set of features offered by transport protocols {{?RFC8923}}. If that minimal set is updated or expanded in the future, the Transport Services API ought to be extended to match.
```

Author: I originally meant "if necessary" to say that the minset change might not require an API change, but I can remove.
